### ISSUE TITLE
Rounding problems with small numbers in visibility_degrid

### ISSUE THREAD

**Issue by adampbeardsley** [EoRImaging/FHD#62]
This bug was discovered when troubleshooting a pyuvdata issue (https://github.com/HERA-Team/pyuvdata/issues/89). I'm looking at lines 66-75 in visibility_degrid.pro. Take, for example, a baseline with `ycen=-1e-20`. The `y_offset` will evaluate to 0, and `ymin` will evaluate to `elements/2.-(psf_dim/2.-1)-1`, when they should evaluate to `psf_resolution-1` and `elements/2.-(psf_dim/2.-1)-1`. 
The result is that the kernel for this visibility is offset by nearly one pixel in the uv plane. This, I believe, is what was causing the jumps in @aelanman 's waterfalls in the pyuvdata issue. One is pasted below for reference.
![image](https://cloud.githubusercontent.com/assets/4752917/20451513/74159610-adb7-11e6-870f-575937af803c.png)
This is an E-W baseline, phased to zenith at halftime of the observation (y-bin=28). Because the data is phased that way, the v coordinate is ~-1e-20 for that exact time, and it sees this bug.

**Comment by aelanman** [comment]
Has this been fixed? It may be related to a new problem I'm getting.

**Comment by isullivan** [comment]
I don't think it has been fixed yet. Could you describe the new problem you are getting?

**Comment by aelanman** [comment]
I'm seeing visibilities that should be redundant falling into small groups that are identical. That is, given a collection of perfectly redundant baselines, there are two or three subgroups of these baselines that have identical visibilities, but the subgroups differ from each other. For east-west baselines, the difference is only in the first time step, which reminded me of this issue so I thought it might be relevant. It might be a pyuvdata issue instead.
